rpm/readline.spec

```diff
@@ -18,10 +24,25 @@ of previously-entered command lines for recalling or editing those
 lines, and for performing csh-like history expansion on previous
 commands.
 
+
+%package -n libreadline%{rl_major}
+Summary:        The Readline Library
+Group:          System/Libraries
```

Comment:
Skip.

Reply:
We follow fedora's packaging, and there it is just readline and this is cosmetic so lets not do tihs.